Component "mahogany/executable" not found.

I got this when I was trying to run `make`. I already have all depenencies. How do I fix this ? Thanks.

```
[lidg ] mahogany $ make
ninja -C /home/lidg/cmp/mahogany/build/heart
ninja: Entering directory `/home/lidg/cmp/mahogany/build/heart'
[342/342] Linking target example/heart
# FIXME?: move the api headers into a separate directory and just use those instead of calling install:
ninja -C /home/lidg/cmp/mahogany/build/heart install > /home/lidg/cmp/mahogany/build/install_output.txt
sbcl --non-interactive --load  build-mahogany.lisp
This is SBCL 2.2.9.debian, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
While evaluating the form starting at line 17, column 0
  of #P"/home/lidg/cmp/mahogany/build-mahogany.lisp":
Unhandled ASDF/FIND-COMPONENT:MISSING-COMPONENT in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                             {10048E81B3}>:
  Component "mahogany/executable" not found

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10048E81B3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK Component "mahogany/executable" not found #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* Component "mahogany/executable" not found)
2: (INVOKE-DEBUGGER Component "mahogany/executable" not found)
3: (ERROR ASDF/FIND-COMPONENT:MISSING-COMPONENT :REQUIRES "mahogany/executable")
4: ((:METHOD ASDF/OPERATE:OPERATE (SYMBOL T)) ASDF/OPERATE:BUILD-OP "mahogany/executable") [fast-method]
5: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> ASDF/OPERATE:BUILD-OP "mahogany/executable")
6: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
7: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<FUNCTION (LAMBDA NIL :IN ASDF/OPERATE:OPERATE) {10025C6E9B}> :OVERRIDE NIL :KEY NIL :OVERRIDE-CACHE NIL :OVERRIDE-FORCING NIL)
8: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/OPERATE:BUILD-OP "mahogany/executable") [fast-method]
9: (ASDF/OPERATE:MAKE "mahogany/executable")
10: (SB-INT:SIMPLE-EVAL-IN-LEXENV (ASDF/OPERATE:MAKE "mahogany/executable") #<NULL-LEXENV>)
11: (EVAL-TLF (ASDF/OPERATE:MAKE "mahogany/executable") 2 NIL)
12: ((LABELS SB-FASL::EVAL-FORM :IN SB-INT:LOAD-AS-SOURCE) (ASDF/OPERATE:MAKE "mahogany/executable") 2)
13: ((LAMBDA (SB-KERNEL:FORM &KEY :CURRENT-INDEX &ALLOW-OTHER-KEYS) :IN SB-INT:LOAD-AS-SOURCE) (ASDF/OPERATE:MAKE "mahogany/executable") :CURRENT-INDEX 2)
14: (SB-C::%DO-FORMS-FROM-INFO #<FUNCTION (LAMBDA (SB-KERNEL:FORM &KEY :CURRENT-INDEX &ALLOW-OTHER-KEYS) :IN SB-INT:LOAD-AS-SOURCE) {10023DA68B}> #<SB-C::SOURCE-INFO {10023DA653}> SB-C::INPUT-ERROR-IN-LOAD)
15: (SB-INT:LOAD-AS-SOURCE #<SB-INT:FORM-TRACKING-STREAM for "file /home/lidg/cmp/mahogany/build-mahogany.lisp" {10023D9183}> :VERBOSE NIL :PRINT NIL :CONTEXT "loading")
16: ((LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) #<SB-INT:FORM-TRACKING-STREAM for "file /home/lidg/cmp/mahogany/build-mahogany.lisp" {10023D9183}> NIL)
17: (SB-FASL::CALL-WITH-LOAD-BINDINGS #<FUNCTION (LABELS SB-FASL::LOAD-STREAM-1 :IN LOAD) {7C63A3DFF97B}> #<SB-INT:FORM-TRACKING-STREAM for "file /home/lidg/cmp/mahogany/build-mahogany.lisp" {10023D9183}> NIL #<SB-INT:FORM-TRACKING-STREAM for "file /home/lidg/cmp/mahogany/build-mahogany.lisp" {10023D9183}>)
18: (LOAD #P"build-mahogany.lisp" :VERBOSE NIL :PRINT NIL :IF-DOES-NOT-EXIST :ERROR :EXTERNAL-FORMAT :DEFAULT)
19: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:LOAD . "build-mahogany.lisp") (:QUIT)))
20: (SB-IMPL::TOPLEVEL-INIT)
21: ((FLET SB-UNIX::BODY :IN SB-IMPL::START-LISP))
22: ((FLET "WITHOUT-INTERRUPTS-BODY-3" :IN SB-IMPL::START-LISP))
23: (SB-IMPL::%START-LISP)

unhandled condition in --disable-debugger mode, quitting
make: *** [Makefile:18: /home/lidg/cmp/mahogany/build/mahogany] Error 1
[lidg ] mahogany $ 
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,4 +14,4 @@ RUN curl -O https://beta.quicklisp.org/quicklisp.lisp \
 COPY . .
 
 # RUN git submodule init
-RUN make test
+RUN make && make test

--- a/build-mahogany.lisp
+++ b/build-mahogany.lisp
@@ -6,7 +6,7 @@
 ;; to be used instead of anything that asdf can find in our environment
 (asdf:initialize-source-registry
  `(:source-registry
-   (:directory ,(merge-pathnames (uiop/os:getcwd) #P"src"))
+   (:directory ,(uiop/os:getcwd))
    (:tree ,(merge-pathnames (uiop/os:getcwd) #P"dependencies"))
    ;; Use whatever the user has configured in their environment to find the rest.
    :inherit-configuration))

